File: number-of-good-pairs.kt
class Solution {
  fun numIdenticalPairs(nums: IntArray): Int {
    var count = 0
    for (i in 0..nums.size - 1) {
      for (j in i..nums.size - 2) {
       if (nums[i] == nums[j + 1]) count++   
      }
    }
    return count
  }
}

File: number-of-good-pairs.py
class Solution:
  def numIdenticalPairs(self, nums: List[int]) -> int:
    count = 0
    inputLength = len(nums)
    for i in range (0, inputLength):
      for j in range (i, inputLength - 1):
        if (nums[i] == nums[j + 1]): count += 1
    return count